package.json

```diff
@@ -108,7 +108,7 @@
     "react-animations": "^0.1.0",
     "react-dom": "16.9.0",
     "react-icons": "3.7.0",
-    "react-intl": "^2.4.0",
+    "react-intl": "3.3.2",
```

Comment:
you are also dropping the `^` here, why?

Reply:
This means that the dependency is specified more strictly which I guess is not a bad idea.

Reply:
`yarn` adds the ^ by default when you don't specify a version when installing. I had specified `3.3.2` in this case.

I am not sure if it is a good idea always to keep it this strict. e.g for well maintained packages, we might miss out on security updates or bugfix patches unless we actively commit those version upgrades in our repo. If we say `react: ~16.9.0` will ensure that we need not update the dependency until we want to switch to `v17`

Reply:
I found this very useful for a quick reference: https://semver.npmjs.com/